dbt/adapters/sqlserver/__version__.py

```diff
@@ -0,0 +1 @@
+version = '0.18.0b2'
```

Comment:
@mikaelene @jtcohen6 any guidance on what this version should be? We'll prolly cut a release soon-ish that incorporates all of these recent PRs.... 

do we include the most recent patch release (`.1`) number? what does the `b` mean?
```suggestion
version = '0.18.1b0'
```

Reply:
I have not done any 0.18.1 release yet. If we fix the snapshot-check-strategy and make sure all connections variations work. Why not 0.18.1? 😀. That is a dbt version that it works well with